src/Classes/Options/Background/Background.ts

```diff
@@ -0,0 +1,46 @@
+import type { IBackground } from "../../../Interfaces/Options/Background/IBackground";
+import type { RecursivePartial } from "../../../Types/RecursivePartial";
+import { Color } from "../Particles/Color";
+import type { IColor } from "../../../Interfaces/Options/Particles/IColor";
+
+export class Background implements IBackground {
+    color?: IColor | string;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The class property 'color' must be marked either 'private', 'public', or 'protected'](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5274115)